save_selected_images.py

```diff
@@ -0,0 +1,51 @@
+"""
+    This script shows one by one each image within a folder, by natural order,
+    and save selected ones typing "s" key in "selected" folder
+"""
+
+
+import cv2
+import glob
+import os
+from natsort import natsorted
+import argparse
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '-images_path',
```

Comment:
it hsould be `--images_path`, you forgot an extra `-`
It has been done correctly in the other argument, `--scale`

Reply:
ok. Two options, one "-" with letter, and two for the word.